tests/test_is_output_correct.py

```diff
@@ -24,9 +24,15 @@
         (5, 5.0),
         ("1/2", 0.5),
         ("\\frac{1}{2}", 0.5),
+        ("\\frac {1}{2}", 0.5),
         ("x^2+2x+1", "x^2 + 2*x + 1"),
         ("x^2+2x+1", "x^2 + 2*x - (-1)"),
         ("x^2+2x+1", "2x+ 1+x^2"),
+        ("17\\text{ any text}", "17"),
+        ("\$10", "10"),
+        ("10%", "1/10"),
+        ("56,\\!01,\\!78,\\!95,\\!760", "56017895760"),
+        ("any text=12", "12"),
```

Comment:
I'm not sure we necessarily want to evaluate this as correct. The model should instead put boxed around a number only